Polimorfism/Polimorfism/Angajat.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Polimorfism
+{
+    internal class Angajat
+    {
+        public string Nume { get; private set; }
+
+        public DateTime DataAngajarii { get; private set; }
+
+        public Angajat(string nume, DateTime dataAngajarii) 
+        {
+            Nume = nume;
+            DataAngajarii = dataAngajarii;
+        }
+
+        public virtual double CalculeazaSalariu()
+        {
+            return 10000;
```

Comment:
Ar fi bine sa scoti asta intr-o constanta aparte, sa fie clar de unde vine 10000 (ca e salariul minim).
https://refactoring.guru/replace-magic-number-with-symbolic-constant#:~:text=A%20magic%20number%20is%20a,to%20change%20this%20magic%20number.